packages/functions/src/api/error.ts

```diff
@@ -146,7 +148,7 @@ export function _errorForResponse(
       }
       // TODO(klimt): Add better default descriptions for error enums.
       // The default description needs to be updated for the new code.
-      description = status;
+      description = status as string;
```

Comment:
@bklimt Can `status` be anything other than string (`description` is defined as a string)? I'm confused as we check if `status` is a string at line 142.

Reply:
As far as the API is concerned, it's supposed to be a string, but this is the code that should be enforcing that, not assuming it. It looks like this line should be moved up into the if-statement above.